worker/runtime/container.go

```diff
@@ -191,11 +192,30 @@ func (c *Container) Attach(pid string, processIO garden.ProcessIO) (process gard
 	}
 
 	cioOpts := containerdCIO(processIO, false)
-	ioAttach := c.ioManager.Attach(c.Handle(), pid, cio.NewAttach(cioOpts...))
 
-	proc, err := task.LoadProcess(ctx, pid, ioAttach)
-	if err != nil {
-		return nil, fmt.Errorf("load proc: %w", err)
+	// Retry LoadProcess - exec may not be initialized yet under load.
+	const maxRetries = 5
+	var proc containerd.Process
+	var lastErr error
+
+	for attempt := 0; attempt < maxRetries; attempt++ {
```

Comment:
nit: can use `range` instead of c-style for loop

```suggestion
	for attempt := range maxRetries {
```